### PR TITLE
Add BigSim and Creating Network Config File sections to the user guide

### PR DESCRIPTION
**Pull request by nightlark**
This PR adds on to the changes in https://github.com/LLNL/TraceR/pull/24, and should be merged after

### Description
Moves most of the UserWriteUp.txt contents to the RTD user guide section on generating traces using Charm++/BigSim , creating the network config file, and tweaks the formatting some more. https://tracer-ryan-test.readthedocs.io/en/docs-update-userwriteup/userguide.html has a build of the docs with these changes.

### Proposed changes
- Add section to RTD user guide on generating traces using Charm++/BigSim
- Add section to RTD user guide on creating the network config file
- Some more small formatting tweaks

### CODE REVIEW

File: docs/userguide.rst [hunk 1]
```diff
@@ -12,41 +12,248 @@ optimistic mode::
...
 
 If you do not intend to create global or per-job map files, you can use ``NA``
 instead of them.
 
-See below to generate global or per-job map files.
+Sample TraceR config files can be found in examples/jacobi2d-bigsim/tracer_config (BigSim) or examples/stencil4d-otf/tracer_config (OTF)
+
+See `Creating the job placement file`_ below for how to generate global or per-job map files.
+
+Creating the network (CODES) configuration file
```
Comment (nightlark) [inline docs/userguide.rst]: A name to use for this file needs to be determined -- simulated system configuration?

**Comment by nightlark** [comment]
I figured out the reason why the options list/table looks different now -- the `breathe` python version 4.13.0.post0 update on April 23 changed to require to Sphinx 2.0; pip then automatically updates Sphinx (default used by RTD is 1.8.5), which results in an option list that looks different.

**Comment by bhatele** [comment]
Is this ready to be merged?

**Comment by nightlark** [comment]
This is ready to be merged, unless you want the version of sphinx ReadTheDocs used pinned to the older one to avoid the TraceR args table style change.

**Pull request merged**